developer-docs-site/docs/guides/basics-life-of-txn.md

```diff
@@ -157,12 +157,15 @@ A client submits a transaction to the REST service of an Aptos fullnode.
 
 ### 2. REST Service → Mempool
 
-The REST service forwards the transaction to a validator fullnode, which then sends it to validator node V<sub>X</sub>'s mempool. The mempool will accept the transaction T<sub>N</sub> only if the sequence number of T<sub>N</sub> is greater than or equal to the current sequence number of the sender's account (note that the transaction will not be passed to consensus until the sequence number matches the sequence number of the sender’s account).
+The REST service of the fullnode puts the transaction in its mempool.
```

Comment:
I'll move the acceptance part back to this section. If the transaction is old, mempool rejects it here, and the REST service will tell the user about it